### PR TITLE
[JENKINS-59211] Enable release-drafter

### PR DESCRIPTION
**Pull request by varyvol**
[JENKINS-59211](https://issues.jenkins-ci.org/browse/JENKINS-59211)

Enable Release Drafter through GitHub Actions.

### CODE REVIEW

**Review by aheritier — COMMENTED** [review]
I have a doubt @varyvol 
I still have the `.github/release-drafter.yml` config file in Advisor. I think it's required to configure the behavior, extends what is preconfigured at the jenkins level and set the tag template.
The workflow file is only to activate the GitHub action to execute it because the server which was doing it is now stopped (or is performing very badly)

**Comment by varyvol** [comment]
@aheritier let's restore it just in case.

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.adoc
 _extends: .github
-tag-template: support-core-$NEXT_MINOR_VERSION
\ No newline at end of file
+tag-template: support-core-$NEXT_MINOR_VERSION
```
Comment (Dohbedoh) [inline .github/release-drafter.yml:3]: Not sure about that space, I'd rather remove it :)

```suggestion
tag-template: support-core-$NEXT_MINOR_VERSION
```

**Review by Dohbedoh — APPROVED** [review]
Yeah I think that the `.github/release-drafter.yml` is required for the configuration of the drafter. 
The `.github/workflows/release-drafter.yml` is a way to enable the drafter GitHub Action from code (as opposed to do it manually in GitHub).

File: .github/release-drafter.yml (hunk 1 above)
Comment (varyvol) [reply .github/release-drafter.yml]: Done!

**Review by Dohbedoh — APPROVED** [review]

**Review by aheritier — APPROVED** [review]
LGTM

**Pull request merged**